remsfal-services/docker-compose.yml

```diff
@@ -4,17 +4,21 @@ include:
 services:
   kafka-broker:
     image: apache/kafka:latest
+    hostname: kafka-broker
     container_name: kafka-broker
     ports:
-      - 9092:9092
+      - "9092:9092"
     environment:
       KAFKA_NODE_ID: 1
       KAFKA_PROCESS_ROLES: broker,controller
-      KAFKA_LISTENERS: PLAINTEXT://:9092,CONTROLLER://:9093
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://:9092
+      KAFKA_LISTENERS: INTERNAL://0.0.0.0:29092,EXTERNAL://0.0.0.0:9092,CONTROLLER://0.0.0.0:9093
```

Comment:
Why is this complex configuration requiered? See https://hub.docker.com/r/apache/kafka